### PR TITLE
[FIX] DCOS-2884 Preserve both stdout and stderr calling "--help" on the subcommand

### PR DESCRIPTION
**Pull request by YuraTolochkevich**

### CODE REVIEW

File: cli/dcoscli/help/main.py [hunk 1]
```diff
@@ -112,9 +114,9 @@ def _help_command(command):
...
-        emitter.publish(stderr.decode("utf-8"))
-        return 1
-    emitter.publish(stdout.decode("utf-8"))
-    return 0
+    executable = subcommand.command_executables(command)
+
+    subproc = Popen([executable,  command] + ['--help'],
+                    stderr=PIPE)
+
+    return analytics.wait_and_track(subproc)
```
Comment (jsancio) [inline cli/dcoscli/help/main.py]: We don't need to call this through the analytics code path. The `dcos` program will do this for us. We should be able to just:
```
 return subprocess.call([executable, command, '--help'])
```

That would inherit `dcos help`'s stdout, stdin, stderr which is what we want.

**Comment by jsancio** [comment]
Thanks! I left you one comment.

**Comment by jsancio** [comment]
@YuraTolochkevich I reviewed this PR and left you one comment. Thanks.

**Comment by YuraTolochkevich** [comment]
@Thanks, fixed!

**Comment by jsancio** [comment]
LGTM!

**Pull request merged**